dbt_server/logging.py

```diff
@@ -1,17 +1,16 @@
-import io
 import json
 import logging
+import uuid
 import os
 from dataclasses import dataclass
 from datetime import datetime
 from typing import Optional
 
-import logbook
-import logbook.queues
-
-from dbt.events.functions import STDOUT_LOG, FILE_LOG
-import dbt.logger as dbt_logger
+from dbt.events.functions import EVENT_MANAGER
```

Comment:
@peterallenwebb  we probably want to move EVENT_MANAGER into a contextvar for this kind of usecase? If we plan to have concurrent request being handled by the same dbt-server container

Reply:
@ChenyuLInx Thanks for opening https://github.com/dbt-labs/dbt-core/issues/6399! Love to see it